pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -276,6 +322,49 @@ func (n *clusterManagerController) sync(ctx context.Context, controllerContext f
 		currentGenerations = append(currentGenerations, currentGeneration)
 	}
 
+	// Try to load ca bundle from configmap
+	// If the configmap is found, populate it into configmap.
+	// If the configmap not found yet, skip this and apply other resources first.
+	caBundle := "placeholder"
+	configmap, err := n.configMapLister.ConfigMaps(clusterManagerNamespace).Get(caBundleConfigmap)
+	switch {
+	case errors.IsNotFound(err):
+		// do nothing
+	case err != nil:
+		return err
+	default:
+		if cb := configmap.Data["ca-bundle.crt"]; len(cb) > 0 {
+			caBundle = cb
+		}
+	}
+	encodedCaBundle := base64.StdEncoding.EncodeToString([]byte(caBundle))
+	config.RegistrationAPIServiceCABundle = encodedCaBundle
+	config.WorkAPIServiceCABundle = encodedCaBundle
+
+	// Apply apiservice files to hub cluster.
+	apiserviceResults := helpers.ApplyDirectly(
```

Comment:
This is introduced by the previous PR https://github.com/open-cluster-management-io/registration-operator/pull/179

The following logic is based on a fact that the namespace is all ready existed.
``` go
        caBundle := "placeholder"
	configmap, err := n.configMapLister.ConfigMaps(clusterManagerNamespace).Get(caBundleConfigmap)
	switch {
	case errors.IsNotFound(err):
		// do nothing
	case err != nil:
		return err
	default:
		if cb := configmap.Data["ca-bundle.crt"]; len(cb) > 0 {
			caBundle = cb
		}
	}
```

After we support the customized installed namespace name, we found sync always failed because before we apply static files to create the namespace, it returns err when trying to get the configmap.

So we pick the apiservice apply out and do it later after namespace is created.

I will add comments on it, but we may need to refactor this part to reduce the complexity.